layouts/docs/single.html

```diff
@@ -16,8 +16,9 @@
                 {{ partial "docs/toc.html" . }}
             </div>
 
-            <div class="md:w-7/12">
+            <div class="md:w-8/12">
                 <h1>{{ .Title }}</h1>
+                {{ partial "docs/gs-note.html" . }}
```

Comment:
Instead of doing it like this, and having to conditionally detect the path in `gs-note.html` to determine if it should be rendered or not, I think it'd be better as a shortcode. https://gohugo.io/templates/shortcode-templates/

Basically, move `gs-note.html` into `layouts/shortcodes` and remove the `{{ if ... }}`

Then at the top of `docs/reference/install.md` and `docs/reference/tutorials/_index.md` add the following:

```
{{< gs-note >}}
```

Consider renaming the shortcode from `gs-note.html` to `get-started-note.html`.

Reply:
Yeah, it occurred to me last night that I should have used a shortcode for this instead. And noted on the rename. :)